cc-ansible

```diff
@@ -365,15 +356,7 @@ kolla_args+=(--globals="$CONFIG_DIR/globals.yml")
 kolla_args+=(--passwords="$CONFIG_DIR/passwords.yml")
 kolla_args+=(--extra node_custom_config="$CONFIG_DIR/node_custom_config")
 kolla_args+=(--extra cc_ansible_site_dir="$CONFIG_DIR")
-# Override python interpreter to point to virtualenv (but only if we're
-# not performing the bootstrap, which is responsible for setting up the
-# virtualenv in the first place!)
-if [[ ! "${POSARGS[*]}" =~ bootstrap-servers ]]; then
-  # Run a quick ad-hoc command to figure out the derived value of the
-  # 'virtualenv' variable, as it can differ by site.
-  virtualenv=$(get_defined_var virtualenv)
-  kolla_args+=(--extra ansible_python_interpreter="$virtualenv/bin/python")
```

Comment:
This point of this code is to ensure that Ansible modules _on the target host_ run inside _the virtualenv created by Kolla._ That's a different type of virtualenv than the rest of this patch addresses. Did you test this on an environment where VIRTUAL_ENV on the deploy node != VIRTUAL_ENV on the target?